index.js

```diff
@@ -222,7 +233,9 @@ module.exports = (function () {
                 return retry_loop();
               }
 
-              save_response_in_cache(options.method, options.url, options.auth.bearer, res, body);
+              if (res.statusCode >= 200 && res.statusCode < 300){
+                save_response_in_cache(options.method, options.url, options.auth.bearer, res, body);
```

Comment:
The use case I envision is the client getting the url for a route wrong and then spamming the service with way more calls since every request could potentially have the wrong url. I agree that would have to be configurable (defaulting to off), and that we don't really need it now anyway